third_party/fflate/README.md

```diff
@@ -0,0 +1,6 @@
+# fflate v0.8.2 gunzipSync only
+- Taken from https://github.com/101arrowz/fflate
+- Pre-transpiled to JS using tsc
+- Removed all option parameters, dictionary and user buffer are unused
+- Inline expand functions that are only used once
```

Comment:
I'm not sure I would worry too much about these two.

Closure compiler can inline stuff if its possible.   Lets just start by getting something that works and is as close to upstream as possible for easy maintenance.

Reply:
Sure! I just want to strippppppppp the size downnnn, but I agree, let's make it work first

Reply:
BTW I think it would be good to commit both the `.ts` source and `.js` file as part of this change.

Can you include here in the readme the exact command you run to convert the `.ts` to `.js`?

Reply:
Sure!